### ISSUE TITLE
placeholder view Unicode character encoding error

### ISSUE THREAD

**Issue by lc-1010** [charmbracelet/bubbles#420]
**Describe the bug**
 `Placeholder` View renders got Unicode character encoding error

**Setup**
use `textinput.Model`set   Unicode character is the Placeholder 
```
inputs= textinput.New()
 inputs.Placeholder = "馹ABC"
...
m.input.View()
...
 ���ABC
```

**Comment by lc-1010** [comment]
You're welcome! I'm glad to hear that you were able to reproduce the issue. 

Whenever you have a moment ,could you kindly review and consider merging my [PR](https://github.com/charmbracelet/bubbles/pull/421) ? Thank you for your time.
 @meowgorithm

**Comment by meowgorithm** [comment]
Oh brilliant, I totally missed that you have have a patch. Let's get this done!

**Comment by meowgorithm** [comment]
Fixed by #421 🙌
